### ISSUE TITLE
"The name 'deepspeech/logits:0' refers to a Tensor which does not exist. The operation, 'deepspeech/logits', does not exist in the graph."

### ISSUE THREAD

**Issue by 912602337** [ashawkey/RAD-NeRF#54]
您好，https://github.com/osmr/deepspeech_features/releases/download/v0.0.1/deepspeech-0_1_0-b90017e8.pb.zip在这个地址手动下载的模型，为什么显示图里没有呢

**Comment by OceanTan** [comment]
> 您好，[https://github.com/osmr/deepspeech_features/releases/download/v0.0.1/deepspeech-0_1_0-b90017e8.pb.zip在这个地址手动下载的模型，为什么显示图里没有呢](https://github.com/osmr/deepspeech_features/releases/download/v0.0.1/deepspeech-0_1_0-b90017e8.pb.zip%E5%9C%A8%E8%BF%99%E4%B8%AA%E5%9C%B0%E5%9D%80%E6%89%8B%E5%8A%A8%E4%B8%8B%E8%BD%BD%E7%9A%84%E6%A8%A1%E5%9E%8B%EF%BC%8C%E4%B8%BA%E4%BB%80%E4%B9%88%E6%98%BE%E7%A4%BA%E5%9B%BE%E9%87%8C%E6%B2%A1%E6%9C%89%E5%91%A2)

你好解决了吗？The name 'deepspeech/logits:0' refers to a Tensor which does not exist. The operation, 'deepspeech/logits', does not exist in the graph.

**Comment by Suryavf** [comment]
In the path 'data_utils/deepspeech_features/deepspeech_features.py' change `deepspeech/logits:0` to `logits:0`.
